src/sdg_hub/flows/qa_generation/document_grounded_qa/enchance_multi_summary_qa/detailed_summary/flow.yaml

```
@@ -0,0 +1,165 @@
+      format_as_messages: true
+
+  - block_type: LLMChatBlock
+    block_config:
+      block_name: gen_detailed_summary
+      input_cols: summary_prompt
+      output_cols: summary
+      max_tokens: 4096
+      temperature: 0.7
+      n: 50
+      async_mode: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)